src/loaders/lottie/tvgLottieParser.h

```diff
@@ -39,7 +39,7 @@ struct LottieParser : LookaheadParserHandler
     bool apply(LottieSlot* slot);
     const char* sid(bool first = false);
     void captureSlots(const char* key);
-    template<LottieProperty::Type type = LottieProperty::Type::Invalid> void registerSlot(LottieObject* obj);
+    template<LottieProperty::Type type = LottieProperty::Type::Invalid> void registerSlot(LottieObject* obj, char* sid = nullptr);
```

Comment:
instead of the optional sid, could you please change the prototype has `sid` as mandatory?
you can also change the callers such as like:
`registerSlot<type>(obj, getStringCopy());` //510 line
`registerSlot<LottieProperty::Type::ColorStop>(gradient, getStringCopy());   //765 line`